test/unittests/test_sbd.py

```diff
@@ -116,15 +116,19 @@ def test_get_sbd_watchdog_timeout(self, mock_get):
         mock_get.assert_called_once_with("SBD_WATCHDOG_TIMEOUT")
 
     @mock.patch('crmsh.utils.service_is_active')
-    def test_get_stonith_watchdog_timeout_return(self, mock_active):
+    @mock.patch('crmsh.sbd.SBDTimeout.get_sbd_watchdog_timeout')
+    def test_get_stonith_watchdog_timeout_return(self, mock_get_sbd_watchdog_timeout, mock_active):
+        mock_get_sbd_watchdog_timeout.return_value = 10
```

Comment:
Why does this backport use a different test case?

Reply:
Cause there are lots of conflicts while cherry-pick
Then I decided to change the unit test part separately